packages/core/src/crypto/__tests__/JwsService.test.ts

```diff
@@ -18,7 +18,7 @@ describe('JwsService', () => {
     const config = getAgentConfig('JwsService')
     wallet = new IndyWallet(config)
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    await wallet.initialize(config.walletConfig!)
+    await wallet.create({ ...config.walletConfig!, keepOpenAfterCreate: true })
```

Comment:
Isn't it a parameter of the `create` method rather than `walletConfig`? Those flag params are harder to understand and use, we're already using them quite a lot. I'm thinking if `createAndOpen` wouldn't be better. No strong opinion just small feedback.